### DISCUSSION TITLE
arbitration lost - errors

### DISCUSSION THREAD

**Discussion by ghhausi** [john30/ebusd#1065] [Q&A]
Good morning,
1st: my ebusd is running quite well (at least with the parameters I need)
But (almost) **every day** in the morning at **the same time** (=06:27:30) I get "arbitration lost" errors.2023-11-23 06:27:29.546 _

> [bus error] poll hmu BuildingCircuitPumpPower failed: ERR: arbitration lost
> 2023-11-23 06:27:29.546 [bus error] send to 08: ERR: arbitration lost, retry
> 2023-11-23 06:27:29.547 [bus error] send to 08: ERR: arbitration lost
> 2023-11-23 06:27:29.547 [bus error] send message part 0: ERR: arbitration lost
> 2023-11-23 06:27:29.549 [bus error] send to 76: ERR: arbitration lost, retry

_
But it continues reading without problems afterwards.
Any idea where I can start solving this problem ?
What is happening at 06:27:30 ?

thanx
Gerhard
PS: 
ebusd options: EBUSD_OPTS="--scanconfig -d ens:/dev/ttyAMA0 --latency=150 --mqtthost=Master --mqttport=1883 --mqttretain --httpport=8889 --port=8888 --mqttint=/etc/ebusd/mqtt-hassio.cfg --htmlpath=/var/www/ebusd/html --loglevel=notice --configpath=/etc/ebusd-config --pollinterval=15"

Vaillant config: aroTherm VWL 75/5 AS, uniTower VWL78/5 IS, VR720, VR921 Gateway
pi@eBusPi:/etc/ebusd-config/vaillant $ ebusctl i
version: ebusd 23.2.23.2
device: /dev/ttyAMA0, enhanced high speed, firmware 1.1[3a0f].1[3a0f]
signal: acquired
symbol rate: 70
max symbol rate: 397
min arbitration micros: 5
max arbitration micros: 887
min symbol latency: 0
max symbol latency: 11
scan: finished
reconnects: 0
masters: 5
messages: 215
conditional: 0
poll: 71
update: 9
address 00: master #1
address 03: master #11
address 05: slave #1, scanned "MF=Vaillant;ID=VR921;SW=2801;HW=5703"
address 08: slave #11, scanned "MF=Vaillant;ID=HMU00;SW=0522;HW=5103", loaded "vaillant/08.hmu.csv"
address 10: master #2
address 15: slave #2, scanned "MF=Vaillant;ID=CTLV2;SW=0514;HW=1104", loaded "vaillant/15.ctlv2.csv"
address 31: master #8, ebusd
address 36: slave #8, ebusd
address 71: master #9
address 76: slave #9, scanned "MF=Vaillant;ID=VWZIO;SW=0902;HW=5103", loaded "vaillant/76.vwzio.SW0902.csv"

**Comment by ghhausi** [comment]
Good morning,
today I had an **issue** with the **ebus** --> signal lost ... **it was dead**.
I unplugged the bus-adapter (for the moment) + power recycled the Vaillant WP + uniTower --> and it is running again (without bus-adapter on the raspi).
Question: Could that be **caused** by the **bus-adapter and ebusd ?**
Log is:
_

> 2023-11-24 08:07:08.061 [update notice] received read hmu State QQ=71: 52;14.9;Normal mode HP on;heating_water
> 2023-11-24 08:07:08.171 [update notice] sent poll-read VMZ HeizstabTemp QQ=31: 6.25
> 2023-11-24 08:07:08.679 [update notice] sent poll-read ctlv2 Hc1ActualFlowTempDesired QQ=31: 0.0
> 2023-11-24 08:07:12.099 [bus error] signal lost
> 2023-11-24 08:07:12.099 [bus error] send to 08: ERR: no signal, give up
> 2023-11-24 08:07:12.099 [bus error] send message part 0: ERR: no signal
> ......
> 2023-11-24 08:08:14.433 [bus error] send message part 0: ERR: no signal
> 2023-11-24 08:08:15.679 [bus notice] device status: resetting
> 2023-11-24 08:08:15.679 [bus notice] re-opened /dev/ttyAMA0
> 2023-11-24 08:08:16.246 [bus notice] device status: reset, supports info
> 2023-11-24 08:08:16.347 [bus notice] device status: extra info: firmware 1.1[3a0f].1[3a0f], jumpers 0x03
> 2023-11-24 08:08:16.460 [bus error] arbitration start error
> 2023-11-24 08:08:16.461 [bus error] send message part 0: ERR: no signal

_
Any ideas ?

BTW: This morning I did not have an arbitration error at 06:27 !

Thanks for your input
br
Gerhard

**Reply by ghhausi** [reply]
Guys - especially @john30 --> could it be that ebusd does something "important" every 24 hours, and an "arbitration lost" error could occur ? because my RasPi (3b+) or the ebusadapter might get too busy ?
Background: I started ebusd the other day in the afternoon .... now I sometimes get "arbitration lost" error exactly at 15:15

thanks for your help
Gerhard

**Reply by john30** [reply]
unless disabled, ebusd once per day checks for version updates. via ssl of course
